microsoft-edge/privacy-whitepaper.md

```diff
@@ -349,14 +357,22 @@ When you start Read aloud, Microsoft Edge uses the [Web Speech API][GithubW3cInc
 
 ## Resolve navigation errors  
 
 If Microsoft Edge detects SSL connection timeouts, certificate errors, or other network issues that may be caused by a captive portal \(for example, a Wi-Fi network at a hotel or airport\), Microsoft Edge sends a requests to `http://edge.microsoft.com/captiveportal/generate_204` and checks the response code.  If the request is redirected to another URL, Microsoft Edge opens the URL in a new tab, assuming that it is a sign-in page.  Requests to the captive portal detection page are a stateless service, requests are not logged, and cookies are not sent or saved.  On Windows platforms, Microsoft Edge uses a Windows captive portal service.  Otherwise, the Microsoft Edge captive portal service is used.  To turn the service off, go to `edge://settings/privacy` and turn off the **Use a web service to help resolve navigation errors** setting.  
 
+## Secure DNS
+
+To navigate to a website, the browser needs to look up the network address (e.g. 93.184.216.34) for the host name (e.g. example.com) that's used in the website's URL. Secure DNS performs this lookup using a service over a HTTPS connection to the DNS service provider, thus protecting the lookups from modification or eavesdropping by attackers on the network.  
+
+By default, your current DNS service provider is used to avoid disruptions to your browsing. Not all service providers offer secure DNS. To avoid any disruption to your browsing, if the secure DNS connection fails, Microsoft Edge will fallback and perform an unencrypted DNS lookup with your current DNS service provider. Microsoft Edge allows you to use a specific secure DNS provider. If a secure DNS provider is chosen, then Microsoft Edge will not fallback to regular DNS lookup if the secure lookup fails. You can choose your secure DNS settings in `edge://setting/privacy`.  
+
+Secure DNS is off by default for managed machines part of an organization but it can be configured using administrative policies. InPrivate browsing uses the secure DNS setting of the profile from which the InPrivate session was launched. Guest mode will always use your current service provider.
```

Comment:
```suggestion
Secure DNS is off by default for managed machines that are part of an organization but it can be configured using administrative policies. InPrivate browsing uses the secure DNS setting of the profile from which the InPrivate session was launched. Guest mode will always use your current service provider.
```